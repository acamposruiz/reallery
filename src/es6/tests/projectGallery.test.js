import React from 'react';
import ProjectGallery from '../projectGallery';
import renderer from 'react-test-renderer';
import state from './mocks/state';

jest.mock('../contentIcons', () => { return 'ContentIcons'; });
jest.mock('react-multimedia-gallery', () => { return 'Gallery'; });

test('ProjectGallery', () => {
    const component = renderer.create(
      <ProjectGallery lng="en" project={state.projects.architecture}/>
    );
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
});