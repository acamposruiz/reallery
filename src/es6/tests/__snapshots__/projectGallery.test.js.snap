// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectGallery 1`] = `
<div
  className="App"
>
  <Gallery
    items={Array []}
    margin={4}
    onClickItem={[Function]}
  />
  <div
    className="loading-msg"
    id="msg-loading-more"
  >
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "<svg class=\\"icon-blank\\" xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 32 32\\"></svg>
",
        }
      }
      style={
        Object {
          "fill": "#d2d2d2",
          "height": 64,
          "width": "85",
        }
      }
    />
    <span
      className="loading-msg-text"
    >
      Loading
    </span>
  </div>
</div>
`;
